### PR TITLE
Improve notification suggestion titles, Add probe suggestions

### PR DESCRIPTION
**Pull request by sapiderman**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Closing #794 
2. Closing #795 
3. Closing #804 

## How did you implement / how did you fix it  
1.  Adding titles to notification objects for better dispaly when suggested
2. Add defaults for url and alerts to help autosuggestion
3. EDIT **Added fix for timeout default**

## How to test  
1. Please see PR #816 on how to test.

### Before
image

### After
https://user-images.githubusercontent.com/964106/180918264-48177618-9002-4e1b-9a72-a1f8ac7c18e1.mp4

### CODE REVIEW

File: monika-config-schema.json [hunk 1]
```diff
@@ -31,20 +33,27 @@
...
-            "description": "The remote URL address to probe"
+            "description": "The remote URL address to probe",
+            "default": "https://api.github.com/zen",
+            "examples": ["https://github.com", "https://httpbin.org/status/200"]
           },
           "timeout": {
+            "title": "Timeout",
             "type": "integer",
-            "description": "Set the timout for the request in seconds",
+            "description": "Set the timeout for the request in seconds",
```
Comment (nicnocquee) [inline monika-config-schema.json:52]: in milliseconds

File: monika-config-schema.json [hunk 2]
```diff
@@ -31,20 +33,27 @@
...
             "default": 10
           },
           "saveBody": {
+            "title": "SaveBody",
```
Comment (nicnocquee) [inline monika-config-schema.json:56]: maybe better: "Save body to db"

**Review by nicnocquee — CHANGES_REQUESTED** [review]

File: monika-config-schema.json (hunk 2 above)
Comment (sapiderman) [reply monika-config-schema.json:56]: this is just title actually, the description is below "should reponse body be save in the database"

**Review by dennypradipta — APPROVED** [review]

**Comment by sapiderman** [comment]
Sorry guys, made a new update: Please check timeout default value (from 10 to 10k ms) to close #804. Thanks!

**Review by nicnocquee — APPROVED** [review]

**Pull request merged**